### ISSUE TITLE
Cannot complete task after editing form during workflow

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#1628]
It's possible to configure that a form is editable during the workflow in the AW Admin.  Currently, we have this configured for the `TS Edit in Workflow` form on edge1.  If you route that form for approval and don't change anything as the approver, your approval works fine.  However, if you update the form and then try to approve, it fails verifying the user's signature:

![image](https://user-images.githubusercontent.com/1377228/36397734-91c75168-1589-11e8-863d-0812fcb4cdf9.png)

```
2018-02-19 15:21:16,666	TP-Processor8	ERROR	DCTMAuthenticationImpl	[DM_DOCBROKER_E_NO_SERVERS_FOR_DOCBASE]error:  "The DocBroker running on host (TSGDEMODCTM2:1489) does not know of a server for the specified docbase (undefined)"
com.documentum.fc.client.DfNoServersException: [DM_DOCBROKER_E_NO_SERVERS_FOR_DOCBASE]error:  "The DocBroker running on host (TSGDEMODCTM2:1489) does not know of a server for the specified docbase (undefined)"
```

Based on the message **The DocBroker running on host (TSGDEMODCTM2:1489) does not know of a server for the specified docbase (undefined)**, it seems that the OC side somehow isn't getting the docbase properly.

A while back, we used to store the Docbase that HPI logged into in the `project-config.js`.  However, that changed a while back so that HPI doesn't pass over anything, and the `DctmAuthenticationImpl` just knows to fall back to the data dictionary repo if no repository is specified.  See OC REV 16517.

It's suspicious however that the functionality works for a normal approve/reject.  It only fails if the approver edits the form.  Perhaps we're actually passing in a docbase name of `undefined` rather than nothing?  It's odd though that it's not doing the same thing if the form was not edited.  I would think it's running the same code...

**Comment by gsteimer** [comment]
Interestingly enough, it does work if I log out and then log back in.  I'm betting hard refreshing would do the same thing.

**Comment by dsubramanian92394** [comment]
Closed with rev #19965
